cachi2/core/errors.py

```diff
@@ -27,3 +50,12 @@ class UnsupportedFeature(Cachi2Error):
 
     The requested feature might be valid, but Cachi2 doesn't implement it.
     """
+
+
+def _friendly_error_msg(reason: str, solution: Optional[str], docs_link: Optional[str]) -> str:
```

Comment:
Any reason not to have the Optional[str] default to `None`?
```
def _friendly_error_msg(reason: str, solution: Optional[str] = None, docs_link: Optional[str] = None) -> str:
```

Reply:
I generally avoid giving helper methods defaults if I'm always going to pass all the params. But in this case I'm not always passing the `docs_link`, updated :+1: 

Not using a default for `solution` because we're not really supposed to be have solution-less "friendly" error messages